app/Filament/Resources/NodeResource/Widgets/NodeCpuChart.php

```diff
@@ -0,0 +1,80 @@
+<?php
+
+namespace App\Filament\Resources\NodeResource\Widgets;
+
+use App\Models\Node;
+use Carbon\Carbon;
+use Filament\Support\RawJs;
+use Filament\Widgets\ChartWidget;
+use Illuminate\Database\Eloquent\Model;
+
+class NodeCpuChart extends ChartWidget
+{
+    protected static ?string $pollingInterval = '5s';
+    protected static ?string $maxHeight = '300px';
+
+    public ?Model $record = null;
+
+    protected function getData(): array
+    {
+        /** @var Node $node */
+        $node = $this->record;
+
+        $cpu = collect(cache()->get("nodes.$node->id.cpu_percent"))
+            ->slice(-10)
+            ->map(fn ($value, $key) => [
+                'cpu' => $value,
+                'timestamp' => Carbon::createFromTimestamp($key)->format('H:i:s'),
```

Comment:
Is the time displayed in the user timezone? Or app timezone (UTC)?
This looks like app timezone.

Reply:
Set to use the users timezone, if that for some reason does not exist it falls back to UTC